docs/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "docs",
-  "version": "5.0.0-beta.0",
+  "version": "5.0.0-beta.1",
```

from your explanation, should this be `5.0.0` because it is private?

---

Good point, it's the version of the root that we use for the documentation:

https://github.com/mui-org/material-ui/blob/deaa9764d938ed3534ed72ff6f1be2fcb796b2de/docs/next.config.js#L3

So we can do:

```suggestion
  "version": "5.0.0-beta.0",
```

---

agree to make it `5.0.0` for private and I hope the next release all the privates does not increment via `lerna version` (is it possible? otherwise we will need to change every time)